daily_read/order_portal.py

```diff
@@ -73,14 +73,17 @@ def process_orders(self, priority, closed_before_in_days=30):
         older_than_cutoff = (pull_date - datetime.timedelta(days=closed_before_in_days)).date()
         for order in self.all_orders:
             # Skip projects closed some time ago or if data is not available
-            if (
-                order["status"] == "closed"
-                and datetime.datetime.strptime(order["history"]["closed"], "%Y-%m-%d").date() >= older_than_cutoff
-            ):
-                continue
-            elif order["identifier"] not in self.projects_data.data.keys():
+            delete_report = False
+            if order["identifier"] not in self.projects_data.data.keys():
                 log.debug(f"Order portal id: {order['identifier']} not found in data fetched from sources")
                 continue
+            elif order["status"] == "closed":
+                # Add 2 day padding for deleting reports from closed projects
+                close_date = datetime.datetime.strptime(order["history"]["closed"], "%Y-%m-%d").date()
+                if close_date < older_than_cutoff - datetime.timedelta(days=2):
```

Comment:
Could you add some more comments here to help the reader figuring out what's going on?

Reply:
That's for this section, not the specific row necessarily

Reply:
Added some more comments now!

Reply:
Thank you!